lti_consumer/translations/locale

```diff
@@ -0,0 +1 @@
+locale
```

Comment:
This file is a symlink to itself? I don't think this is correct.

Reply:
@OmarIthawi I've commited the wrong symlink. Fixing it soon :)

Reply:
Fixed!